Cargo.lock

```diff
@@ -2131,9 +2131,8 @@ dependencies = [
 
 [[package]]
 name = "gilrs"
-version = "0.10.8"
-source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "3f226b8f4d9bc7da93de8efd8747c6b1086409ca3f4b6d51e9a7f5461a9183fe"
+version = "0.10.6"
+source = "git+https://gitlab.com/gilrs-project/gilrs?rev=eafb7f2ef488874188c5d75adce9aef486be9d4e#eafb7f2ef488874188c5d75adce9aef486be9d4e"
```

Comment:
@msub2 Is this specific Git version necessary? Would it be fine to update gilrs to a newer published release?

Reply:
This specific revision is necessary because the currently published version on crates.io does not contain the changes I made to support events for force feedback effect completion. I've been meaning to ask the maintainer what's going on with that, as the last time I talked with them it sounded like it would've landed 2 or 3 releases ago

Reply:
Just left a message on the original gilrs PR, will see what they say